docs/documentation/contributing.md

```diff
@@ -7,7 +7,7 @@ First and foremost thanks to anyone who contributes, very much appreciated.
 - Submit a PR with your change and if there are no comments, changes will be merged in
 - If you're not sure about the change, raise an issue and have a discussion before spending time coding it up
 - Try and make one logical change per PR. That is not make many changes in one PR. Submit multiple PRs instead
-- Java 8 is our target version. If you need anything older than that, we recommend using [Java Faker](https://github.com/DiUS/java-faker) instead.
+- Since Datafaker 2.x, Java 17 is our target version. If you need anything older than that, we recommend using the 1.x versions of Datafaker instead. Note that while the 1.x is stable and free of known bugs, this branch won't receive future updates.
```

Comment:
I might be over thinking here so don't block on it.

But because 2.x is up coming shouldn't this be "For" or "Starting with" instead of "Since"?

Reply:
You're probably right, I'm no native speaker. I'll update it, thanks!